### ISSUE TITLE
[Feature Request] Add Tooltips for SegmentedButton

### ISSUE THREAD

**Issue by ZhulanovAA** [flutter/flutter#128374]
# Issue
When using `SegmentedButton` you might want to add tooltips explaing each provided option. Right now you can either add `Tooltip` around whole `SegmentedButton` which won't allow you to have separate tooltips for each segment or you can add `Tooltip` as a `label` of each `ButtonSegment`, but then only the label of each segment will have tooltip and not the whole segment (which also includes icon and padding around them)
# Proposal
Add `String? tooltip` field for `ButtonSegment` and if present, put `Tooltip` around corresponding segment widget

**Comment by HansMuller** [comment]
CC @chunhtai

**Comment by chunhtai** [comment]
can you wrap tooltip around ButtonSegment.label widget?

**Comment by ZhulanovAA** [comment]
@chunhtai, yes, but there are also (possibly) icon and padding around them, so big portion of the actual button won't be covered by tooltip. So user might hover/long tap some area of the button, expecting tooltip, but not getting it. Idealy, tooltip should be aroung the whole segment, and not just label which takes relatively small portion of its space

**Comment by HansMuller** [comment]
Perhaps we should give `ButtonSegment` a tooltip property. Although we're not planning to do so in the near term, happy to review a PR if you'd like to make a contribution.

**Comment by ZhulanovAA** [comment]
Ok, it seems I've figured out how to do it, so will try to make a PR soon
